pkg/kv/kvnemesis/generator.go

```diff
@@ -500,6 +514,100 @@ func randPutExisting(g *generator, rng *rand.Rand) Operation {
 	return put(key, seq)
 }
 
+func randAddSSTable(g *generator, rng *rand.Rand) Operation {
+	ctx := context.Background()
+
+	sstTimestamp := hlc.MinTimestamp // replaced via SSTTimestampToRequestTimestamp
+	numKeys := rng.Intn(16) + 1      // number of point keys
+	probReplace := 0.2               // probability to replace existing key, if possible
+	probTombstone := 0.2             // probability to write a tombstone
+	asWrites := rng.Float64() < 0.2  // IngestAsWrites
+
+	// AddSSTable requests cannot span multiple ranges, so we try to fit them
+	// within an existing range. This may race with a concurrent split, in which
+	// case the AddSSTable will fail, but that's ok -- most should still succeed.
+	ranges := make([]string, 1, len(g.currentSplits)+1) // first range starts at ""
+	for key := range g.currentSplits {
+		ranges = append(ranges, key)
+	}
+	sort.Strings(ranges)
+
+	var rangeStart, rangeEnd string
+	rangeIdx := rng.Intn(len(ranges))
+	rangeStart = ranges[rangeIdx]
+	if rangeIdx+1 < len(ranges) {
+		rangeEnd = ranges[rangeIdx+1]
+	}
+	inRange := func(key string) bool {
+		return key >= rangeStart && (key < rangeEnd || rangeEnd == "")
```

Comment:
All keys are encoded uints, check the `fk` and `tk` methods. You can do int arithmetic for everything and this is how we do it elsewhere, I would prefer we stick to one pattern because each way of interacting with their keys has its idiosyncrasies and diversity doesn't pay off.

Reply:
Sure, done.